.pre-commit-config.yaml

```diff
@@ -74,3 +74,4 @@ repos:
         args: [--config-file=mypy.ini, --show-column-numbers]
         files: ^airbyte_cdk/
         pass_filenames: true
+        additional_dependencies: ["types-requests", "types-PyYAML"]
```

Comment:
unrelated, but i was having issues getting my pre-commit checks for `mypy` to pass without these included, i'm not sure if others have also observed this